dashboard/src/setupTests.ts

```diff
@@ -30,3 +31,24 @@ mockIntersectionObserver.mockReturnValue({
   disconnect: () => null,
 });
 window.IntersectionObserver = mockIntersectionObserver;
+
+// Disable console.error/warn for some known test warnings
+
+const originalError = console.error.bind(console.error);
+const originalWarn = console.warn.bind(console.warn);
+
+const ignoredMgs = [
+  "Could not create web worker", // monaco uses web workers, but we don't need them for testing
+  "MonacoEnvironment.getWorkerUrl or MonacoEnvironment.getWorker", // monaco uses web workers, but we don't need them for testing
+  "react-tooltip.min.cjs", // react-tooltip complains about some tests without an "act()" wrapper
+];
```

Comment:
While debugging why the dashboard tests were hanging, I added this to ignore some known errors. This way, the output of the `yarn test` is more legible now.